engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/AncientDataMotionStrategy.java

```diff
@@ -486,6 +486,9 @@ public void copyAsync(DataObject srcData, DataObject destData, Host destHost, As
             if (answer != null && !answer.getResult()) {
                 errMsg = answer.getDetails();
             }
+            if (answer == null) {
+                errMsg = "answer is null, set to error for CopyCommandResult";
```

Comment:
I don't understand this error message. Can you make it very clear, please?
for example "no answer from came back from copyCommand, considering it as failed."
preferably even with some data like src, dest, src.type and dest.type